# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/moz.build
# Commit: 85fd9fc9cffa
# Full Hash: 85fd9fc9cffa98039328796d30a2d10386283b3e
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-21 21:39:46
# Regressor Bug: 1742437
# File Overlap Count: 2
# Description:
#   Bug 1742437 - Move module loader files into a central folder; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137532
# ==============================================================================

diff -r 7ecc4f9b20ad -r 85fd9fc9cffa dom/script/moz.build
--- a/dom/script/moz.build	Mon Feb 21 15:39:08 2022 +0000
+++ b/dom/script/moz.build	Mon Feb 21 15:39:08 2022 +0000
@@ -19,37 +19,29 @@
 
 EXPORTS.mozilla.dom += [
     "AutoEntryScript.h",
-    "LoadedScript.h",
     "ModuleLoader.h",
-    "ModuleLoaderBase.h",
-    "ModuleMapKey.h",
     "ScriptDecoding.h",
     "ScriptElement.h",
-    "ScriptKind.h",
     "ScriptLoadContext.h",
     "ScriptLoader.h",
-    "ScriptLoadRequest.h",
     "ScriptSettings.h",
+    "ScriptTrace.h",
 ]
 
 UNIFIED_SOURCES += [
     "AutoEntryScript.cpp",
-    "LoadedScript.cpp",
     "ModuleLoader.cpp",
-    "ModuleLoaderBase.cpp",
-    "ModuleLoadRequest.cpp",
-    "ModuleMapKey.cpp",
     "nsIScriptElement.cpp",
     "ScriptElement.cpp",
     "ScriptLoadContext.cpp",
     "ScriptLoader.cpp",
     "ScriptLoadHandler.cpp",
-    "ScriptLoadRequest.cpp",
     "ScriptSettings.cpp",
 ]
 
 LOCAL_INCLUDES += [
     "/dom/base",
+    "/js/loader",
 ]
 
 include("/ipc/chromium/chromium-config.mozbuild")